sdk/servicebus/service-bus/README.md

```diff
@@ -39,39 +39,25 @@ You also need to enable `compilerOptions.allowSyntheticDefaultImports` in your t
 
 ### Authenticate the client
 
-Interaction with Service Bus starts with an instance of the [ServiceBusClient](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient) class. You can instantiate
-this class using one of the 3 static methods on it
+Interaction with Service Bus starts with an instance of the [ServiceBusClient](https://docs.microsoft.com/en-us/javascript/api/%40azure/service-bus/servicebusclient) class.
```

Comment:
These links will go to Track 1. And some of them might be 404 until we publish. Others will still go 404 when are in Preview 2 and change APIs

In Event Hubs, we used the azure.github.io links while we were in preview and moved to the docs.microsoft.com links only after the APIs settled down.

Reply:
It was also super annoying because I also had to guess I'll swap over to github.io and kill two birds with one stone. :)

Reply:
Swapped links over to what I think the azuresdkdocs.blob.core.windows.net proper URLs will be.